packages/appium/docs/en/guides/managing-exts.md

```diff
@@ -2,4 +2,78 @@
 title: Managing Appium Drivers and Plugins
 ---
 
-TODO
+To do anything useful with Appium, you need to have at least one [Driver](../intro/drivers.md)
+installed, otherwise Appium won't know how to automate anything. And there is a whole
+[Ecosystem](../ecosystem/index.md) of drivers and plugins out there which enable you to get your
+job done with Appium. This guide helps explain how to manage these drivers and plugins. There are
+two basic strategies: using Appium's extension CLI interface, or managing extensions yourself in an
+NPM-based project.
+
+## Using Appium's Extension CLI
+
+With Appium's [Extension CLI](../cli/extensions.md), you let Appium manage drivers and plugins for
+you, and simply use CLI commands to tell Appium which extensions you would like to install, update,
+or remove. Here's an example of how you might install a driver using the CLI:
+
+```bash
+appium driver install xcuitest
+```
+
+This simple command will install the latest version of the [XCUITest
+Driver](https://github.com/appium/appium-xcuitest-driver). The Extension CLI comes with a variety
```

Comment:
```suggestion
This command will install the latest version of the [XCUITest Driver](https://github.com/appium/appium-xcuitest-driver). The Extension CLI comes with a variety
```

Reply:
1. weasel word
2. I don't think you can have a linebreak in a markdown link

Reply:
fair enough on weasel word, but yes you can have a linebreak in a link, i do it all the time